apps/api/src/app/controllers/resources/book.controller.ts

```diff
@@ -14,18 +13,25 @@ import { RESOURCES_ROUTE_PREFIX } from './constants';
 export class BookController {
     constructor(private readonly bookQueryService: BookQueryService) {}
 
+    // @ApiBearerAuth('JWT')
+    // @UseGuards(OptionalJwtAuthGuard)
     @ApiParam(buildByIdApiParamMetadata())
     @ApiOkResponse({ type: BookViewModel })
     @Get(`/:id`)
-    async fetchById(@Res() res, @Param('id') id: unknown) {
-        const searchResult = await this.bookQueryService.fetchById(id);
+    async fetchById(@Request() req, @Res() res, @Param('id') id: unknown) {
+        const searchResult = await this.bookQueryService.fetchById(id, req.user || undefined);
```

Comment:
Either
- update the `OptionalJwtStrategy` to put `user:undefind` on the request body
- or create a `@User()` decorator that handles this
we do not want these little boolean expressions everywhere